### PR TITLE
Add fetch with fallback method

### PR DESCRIPTION
**Pull request by koonopek**

### CODE REVIEW

**Comment by koonopek** [comment]
Published on npm

File: packages/utils/src/common/fetch-with-fallback.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  urls: string[];
+  axiosConfig?: AxiosRequestConfig<any>;
+};
+
+export const fetchWithFallbacks = async <T>(args: AxiosGetRequest) => {
+  const requests = args.urls.map((url) => axios.get<T>(url, args.axiosConfig));
+
+  const response = await Promise.any(requests);
+
+  return response;
```
Comment (cehali) [inline packages/utils/src/common/fetch-with-fallback.ts:13]:
```js
return Promise.any(requests);
```

**Review by hatskier — APPROVED** [review]

File: packages/utils/src/common/fetch-with-fallback.ts (hunk 1 above)
Comment (hatskier) [reply packages/utils/src/common/fetch-with-fallback.ts:13]: I agree, we can also add a return type here (e.g. smth like `Promise<AxiosResponse<T>>` and remove async and await in the function

**Review by hatskier — APPROVED** [review]
Good job!

File: packages/utils/src/common/fetch-with-fallback.ts (hunk 1 above)
Comment (koonopek) [reply packages/utils/src/common/fetch-with-fallback.ts:13]: done

**Pull request merged**